### PR TITLE
Display leaves functionality added for employees

### PR DESCRIPTION
**Pull request by jagdeepbaidwan**
Display leaves functionality added for employees

### CODE REVIEW

**Comment by jagdeepbaidwan** [comment]
Sorry its for admin, admin can see leaves for all the employees as well as for him/her.

**Review by Arshpreet-Singh38 — APPROVED** [review]
Looks Good! It works well.

File: src/employee_management.c [hunk 1]
```diff
@@ -473,6 +473,10 @@ int emp_management(int i,int emp_id)
                 }
                 else if(ch==2)
                 {
+                    int emp_id;
+		    printf("Enter the employee id \n");
+                    scanf("%d",&emp_id);
+                    int k=display_leaves(emp_id);
```
Comment (varinderdhanda) [inline src/employee_management.c]: if Employee does not exist, then dont u think we can show a message that employee id is invalid?

**Review by varinderdhanda — COMMENTED** [review]
It will show all the leaves what if I want to see only particular year's leave assigned to an employee. we can add that functionality.

File: src/employee_management.c (hunk 1 above)
Comment (jagdeepbaidwan) [reply src/employee_management.c]: Yes we can

Comment (jagdeepbaidwan) [reply src/employee_management.c]: I raise new request with changed functionality after this

**Review by puneetsingh3-sysc — APPROVED** [review]
Serves the purpose.

**Pull request merged**